virttest/test_setup.py

```diff
@@ -2423,38 +2424,71 @@ def __init__(self, test, log_level="1", log_file=""):
         self.log_level = log_level
         self.log_file = log_file
         self.test = test
-        self.libvirtd = utils_libvirtd.Libvirtd()
-        self.libvirtd_conf = utils_config.LibvirtdConfig()
-        self.backupfile = "%s.backup" % self.libvirtd_conf.conf_path
+        self.daemons_dict = {}
+        self.daemons_dict["libvirtd"] = {
+            "daemon": utils_libvirtd.Libvirtd("virtqemud"),
```

Comment:
I think here better remove the "virtqemud" to be more clear

Reply:
Here the purpose to use virtqemud is to set ' virtqemud' if modular daemon is enabled, otherwise set 'libvirtd'. 
If we use utils_libvirtd.Libvirtd(), all sub daemons will be used if modular daemon is enabled. So that is not her purpose, I think.